shannon/testnet/client.toml

```diff
@@ -0,0 +1,5 @@
+broadcast-mode = "sync"
+chain-id = "poktroll"
```

Comment:
If changing the chain id in genesis, this also needs to be modified.

Reply:
pocket-beta